pom.xml

```diff
@@ -59,7 +59,7 @@ limitations under the License.
 
         <!-- google-cloud-java related dependency versions -->
         <!-- TODO: check if commons-codec was upgraded to 1.13 in org.apache.httpcomponents from http-client-->
-        <google-cloud.version>0.116.0-alpha</google-cloud.version>
+        <google-cloud.version>0.119.0-alpha</google-cloud.version>
```

Comment:
This is a version of google-cloud-**client** bom. The version 0.120.0-alpha does not have bigtable-related managed dependencies (See https://github.com/googleapis/java-bigtable-hbase/pull/2332).